### ISSUE TITLE
Hyperparameter issues

### ISSUE THREAD

**Issue by giyhub1lili** [GaoShuang98/CVCities#13]
Hello, may I ask if there is a problem with the hyperparameters in the code of University, I found that the hyperparameter options in the two related codes are S2D, there is no hyperparameter of D2S, and after changing the S2D option in the train_universityDS.py to D2S, there is a big deviation between the accuracy and the accuracy you gave。

**Comment by GaoShuang98** [comment]
Hi，I have updated the D2S Training Code, you can try to training again

**Comment by giyhub1lili** [comment]
With the latest parameters, the results of the run are as follows, reaching the highest accuracy on the 12th epoch (may it be due to the difference between the GPU graphics card model?). ）
------------------------------[2025-03-03 11:29:59/Epoch: 12]------------------------------
Epoch: 12, Train Loss = 0.576, Lr = 0.003981

------------------------------[Evaluate]------------------------------
Extract Features:
Compute Scores:
Recall@1: 95.4537 - Recall@5: 98.5603 - Recall@10: 99.0648 - Recall@top1: 99.1256 - AP: 96.1885

**Comment by wincj** [comment]
> 使用最新的参数，运行的结果如下，在第12个epoch达到了最高精度（可能是因为GPU显卡型号差异？） ------------------------------[2025-03-03 11:29:59/Epoch: 12]------------------------------------------ Epoch: 12, Train Loss = 0.576, Lr = 0.003981
> 
> ------------------------------[评估]------------------------------ 提取特征： 计算分数： 召回率@1：95.4537 - 召回率@5：98.5603 - 召回率@10：99.0648 - 召回率@top1：99.1256 - AP：96.1885

hello，Can you share with me the pre trained weights of dinov2 that you are using? Can I add your contact information if you have WeChat

**Comment by wincj** [comment]
> > > 使用最新的参数，运行的结果如下，在第12个epoch达到了最高精度（可能是因为GPU显卡型号差异？） ------------------------------[2025-03-03 11:29:59/Epoch: 12]------------------------------------------ Epoch: 12, Train Loss = 0.576, Lr = 0.003981
> > > ------------------------------[评估]------------------------------ 提取特征： 计算分数： 召回率[@1](https://github.com/1)：95.4537 - 召回率[@5](https://github.com/5)：98.5603 - 召回率[@10](https://github.com/10)：99.0648 - 召回率[@top1](https://github.com/top1)：99.1256 - AP：96.1885
> > 
> > 
> > hello，Can you share with me the pre trained weights of dinov2 that you are using? Can I add your contact information if you have WeChat
> 
> You can download the unregistered version from the dinov2 project page：https://github.com/facebookresearch/dinov2

Wow, thank you for your reply. You are so enthusiastic! May I add your email or other contact information (such as WeChat), as I do not have access to the datasets VIGOR and CVACT used by the author, and I have not received a response from the original author. I hope to receive your help. If possible, could you share these two datasets with me?
If you are still conducting related research, I also hope to collaborate with you! Can you add a contact information
